wiki/Tournaments/OMIC/2019_4K/id.md

```diff
@@ -0,0 +1,345 @@
+---
+tags:
+- O!MIC 4K 2019
+- OMIC
+---
+
+# osu!mania 4K Indonesia Cup 2019
+![O!MIC 4K 2019 Banner](img/logo.png)  
+**osu!mania 4K Indonesia Cup 2019** adalah turnamen solo osu!mania 4K khusus wilayah Indonesia dengan memakai sistem pertandingan Royal Rumble pada babak setelah Kualifikasi, lalu satu lawan satu dengan Double-Elimination dimulai dari Penyisihan Grup.
+
+## Jadwal Turnamen
+| Sesi | Tanggal |
+| --: | :-- |
+| Pendaftaran | 26 Mei - 9 Juni 2019 |
+| Screening | 10 - 16 Juni 2019
+| Kualifikasi | 22 - 23 Juni 2019 |
+| Babak R56 | 29 - 30 Juni 2019 |
+| Penyisihan Grup - Minggu 1 | 6 - 7 Juli 2019 |
+| Penyisihan Grup - Minggu 2 | 13 - 14 Juli 2019 |
+| Babak R16 | 20 - 21 Juli 2019 |
+| Perempat Final | 27 - 28 Juli 2019 |
+| Semi Final | 3 - 4 Juli 2019 |
+| Final | 10 - 11 Agustus 2019 |
+| Grand Final | 17 - 18 Agustus 2019 |
+
+## Hadiah
+| Posisi | Hadiah |
+| :-: | :-- |
+| ![](https://github.com/ppy/osu-wiki/blob/master/wiki/shared/GCrown.png) | 350K IDR + 4 bulan osu!supporter tag + Kode Voucher Google Play 50K IDR + Profile Badge |
+| ![](https://github.com/ppy/osu-wiki/blob/master/wiki/shared/SCrown.png) | 200K IDR + 2 bulan osu!supporter tag |
+| ![](https://github.com/ppy/osu-wiki/blob/master/wiki/shared/BCrown.png) | 150K IDR + 1 bulan osu!supporter tag |
+
+## Daftar Panitia
+| Posisi | Anggota |
+| :-- | :-- |
+| Ketua Panitia | ![](https://github.com/ppy/osu-wiki/blob/master/wiki/shared/flag/ID.gif) [LuxAzure](https://osu.ppy.sh/users/6045757) |
```

Comment:
```suggestion
| Ketua Panitia | ![][flag_ID] [LuxAzure](https://osu.ppy.sh/users/6045757) |
```
For flags in GitHub, we use [ASC styled](https://osu.ppy.sh/help/wiki/Article_Styling_Criteria) format. Replace all flags following ASC format.